### PR TITLE
450 add dtype field to strings array like and pdarray

### PR DESCRIPTION
**Pull request by hokiegeek2**

### CODE REVIEW

File: arkouda/strings.py [hunk 1]
```diff
@@ -89,7 +91,8 @@ def __init__(self, offset_attrib : Union[pdarray,np.ndarray],
             self.ndim = self.offsets.ndim
             self.shape = self.offsets.shape
         except Exception as e:
-            raise ValueError(e)
+            raise ValueError(e)   
+        self.dtype = self.bytes.dtype
```
Comment (mhmerrill) [inline arkouda/strings.py:95]: doesn't this need to change to `self.dtype = np.str_` or something

Comment (hokiegeek2) [reply arkouda/strings.py]: LOL, yeah, git fail on the part of your favorite contractor.

File: arkouda/strings.py [hunk 2]
```diff
@@ -32,6 +32,8 @@ class Strings:
         The rank of the array (currently only rank 1 arrays supported)
     shape : tuple
         The sizes of each dimension of the array
+    dtype : dtype
+        The dtype is np.str_
```
Comment (mhmerrill) [inline arkouda/strings.py:36]: missed this should be `np.str`

Comment (hokiegeek2) [reply arkouda/strings.py]: @mhmerrill good catch, just fixed it

**Review by reuster986 — APPROVED** [review]
Looks good to me!

**Pull request merged**